changelog.md

```diff
@@ -7,162 +7,272 @@ _Changes marked with a :warning: contain potential breaking changes depending on
 ### :rocket: Improvements
 
 - **create-directus-project**
```

Comment:
@rijkvanzanten I believe this changes were introduced by me, but I only merged main with this branch 😐

Reply:
I believe this is the same as last time we talked about, where the changelog/docs weren't linted, but merging it ended up linting it. No worries 👍